### PR TITLE
Remove trailing whitespace

### PR DESCRIPTION
**Pull request by masterleinad**
```
find . -type f \( -name "*.md" -o -name "*.cc" -o -name "*.h" -o -name "*.txt" -o -name "*.cmake" \) -exec sed -i "s/\s\+$//g" {} \;
```

### CODE REVIEW

File: scripts/docker/check_format_cpp.sh [hunk 1]
```diff
@@ -1,75 +1,5 @@
...
-            $clang_format_executable -i $file
-        fi
-    done
-else
-    echo "OK"
-fi
-exit $n_unformatted_files
+./scripts/apply-clang-format || exit $?
+git diff
+git diff-files --quiet
```
Comment (dalg24) [inline scripts/docker/check_format_cpp.sh]: `git diff-files --patch --exit-code`

Comment (masterleinad) [reply scripts/docker/check_format_cpp.sh]: That seems to do the job as well.

**Comment by masterleinad** [comment]
Retest this please.

**Comment by masterleinad** [comment]
Retest this please.

**Comment by dalg24** [comment]
Please cleanup history to re-trigger the tests

**Review by dalg24 — APPROVED** [review]

**Pull request merged**